chpl/chpl-api/e2e/collections/surveillance-report-controller.postman_collection.json

```diff
@@ -0,0 +1,3753 @@
+{
+	"info": {
+		"name": "surveillance-report-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "annualSurveillanceReportTests",
+			"item": [
+				{
+					"name": "GET /surveillance-report/annual  - ONC STAFF",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"var ReqBody= {\r",
+									"        userName:pm.environment.get(\"emailONCStaff\"),password:pm.environment.get(\"passwordONCStaff\")\r",
```

Comment:
We can create a folders for each role and move requests for the role under it. It will be easy to read through plus we can reduce duplicate pre-request script code and move all of pre-request script at folder level.
We can define pre-request script(authorization) code at folder level it self so that we don't have to have those at each request level

Reply:
I like that idea. I wanted to do something to reduce duplication in the script but didn't get a chance to do that. If you strongly suggest to do that in this PR, I can work on it OR we could do that across all the test scripts to improve the overall structure ( preferably when we have no other API aqa tests ticket on the board to avoid the merge conflicts)?

Reply:
I think this is really good candidate as it has many tests covering all roles. If this ticket is not urgent, is it possible to add in here so other future code follow this pattern which would reduce rework. 

Also, It is really good PR. Thank you for covering so many tests.

Reply:
I'll get started on that sometime this afternoon and test it out over the weekend. As far as time sensitivity goes, this ticket is in standard wip and we have some time before next release. We can talk about it during next standup with the team if there are any concerns(hopefully this will be ready-to-go by then and we won't need that..let's see)